jsBasics/lesson22/package.json

```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "test:pw": "playwright test",
+    "test:pw": "playwright test --headed",
+    "test:pwLoggedin": "cp './src/environments/.env.qa' '.env' && source .env && playwright test --grep '@thistest' --headed --project='Google Chrome'",
```

Comment:
this can be re-written to "npm run test:qa-custom-env:headed -- --grep '@thistest' --project='Google Chrome'" to bypass only missing parts of the script, if the main part fits you :)